### PR TITLE
feat(pci.quota): increase quota by buying credits

### PR DESCRIPTION
**Pull request by lizardK**
| Question         | Answer
| ---------------- | ---
| Branch?          | `develop` 
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #MANAGER-6910
| License          | BSD 3-Clause

## Description

Increase PCI quota by buying credits

ref: MANAGER-6910

Signed-off-by: Cyril Biencourt <cyril.biencourt@ovhcloud.com>

## Related
- [x] TRANS-36967

### CODE REVIEW

File: packages/manager/modules/pci/src/projects/project/quota/increase-request/translations/Messages_fr_FR.json [hunk 1]
```diff
@@ -3,6 +3,13 @@
   "pci_projects_project_quota_increase_submit_label": "Confirmer",
   "pci_projects_project_quota_increase_cancel_label": "Annuler",
   "pci_projects_project_quota_increase_content": "Êtes-vous sûr de vouloir augmenter les quotas de votre projet <strong>{{ projectId }}</strong> ?",
+  "pci_projects_project_quota_increase_buy_success_message": "Le bon de commande pour l'ajout de crédits à votre projet est disponible  <a class=\"oui-link_icon\" target=\"_blank\" href=\"{{ billingUrl }}\">ici<span class=\"oui-icon oui-icon-external_link\" aria-hidden=\"true\"></span></a>",
```
Comment (antleblanc) [inline packages/manager/modules/pci/src/projects/project/quota/increase-request/translations/Messages_fr_FR.json:6]: Extra space character found here:

```suggestion
  "pci_projects_project_quota_increase_buy_success_message": "Le bon de commande pour l'ajout de crédits à votre projet est disponible <a class=\"oui-link_icon\" target=\"_blank\" href=\"{{ billingUrl }}\">ici<span class=\"oui-icon oui-icon-external_link\" aria-hidden=\"true\"></span></a>",
```

**Review by antleblanc — APPROVED** [review]

**Review by frenautvh — APPROVED** [review]

**Review by antleblanc — APPROVED** [review]

**Review by Jisay — DISMISSED** [review]

**Comment by ovh-cds** [comment]
CDS Report translator-push#18860.0 ✘  
* 
  * push ✘

**Comment by ovh-cds** [comment]
CDS Report translator-push#18860.1 ✘  
* 
  * push ✘

**Comment by marie-j** [comment]
Rebased on master

**Review by marie-j — APPROVED** [review]

**Review by antleblanc — APPROVED** [review]

**Pull request merged**